regen/mg_vtable.pl

```diff
@@ -538,7 +525,7 @@ BEGIN
     print $vt "  { $funcs }$comma\n";
     print $vt <<"EOH" if $data->{cond};
 #else
-  { 0, 0, 0, 0, 0, 0, 0, 0 }$comma
+  { .svt_get = 0 }$comma
```

Comment:
Can this not be `{}` like the other all-zero `MGVTBL`s were changed to? Or should they also have at least one member initialised? And in that case, could they both be just `{0}`?

Reply:
Or is the difference because `static` vs not?

Reply:
Oh, yes it can probably be the same. I had a few different attempts at appeasing the compiler.

Reply:
Looks like MSVC [doesn't like](https://github.com/Perl/perl5/actions/runs/8333990114/job/22806953844?pr=22086#step:4:3002) `{}` even for `static` definitions, so I think going for `{0}` consistently is the way to go.